sklearn/neighbors/_lof.py

```diff
@@ -317,6 +317,16 @@ def fit(self, X, y=None):
                 self.negative_outlier_factor_, 100.0 * self.contamination
             )
 
+        """
+        Verify if negative_outlier_factor_ values are within acceptable range
+        Novelty must also be false to detect outliers
+        """
```

Comment:
```suggestion
        # Verify if negative_outlier_factor_ values are within acceptable range.
        # Novelty must also be false to detect outliers
```
No need for syntax abuse, use `#` instead :D

Reply:
Thank you for the feedback!